### PR TITLE
vercel-cli 32.5.0

### PR DESCRIPTION
**Pull request by dnhn**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by dnhn** [comment]
```
Error: inreplace failed
  dist/index.js:
    expected replacement of "= getUpdateCommand" with "= async()=>'brew upgrade vercel-cli'"
```

**Comment by SMillerDev** [comment]
```
  vercel-cli
    * Binaries built for a non-native architecture were installed into vercel-cli's prefix.
      The offending files are:
        /opt/homebrew/Cellar/vercel-cli/32.5.0/libexec/lib/node_modules/vercel/node_modules/fsevents/fsevents.node	(x86_64)
```

**Review by chenrui333 — APPROVED** [review]

File: Formula/v/vercel-cli.rb [hunk 1]
```diff
@@ -35,6 +35,7 @@ def install
                 .each { |dir| dir.rmtree if dir.basename.to_s != "#{os}-#{arch}" }
 
     # Replace universal binaries with native slices
+    (node_modules/"fsevents/fsevents.node").unlink if OS.mac? && Hardware::CPU.arm?
```
[the remark this answers is not in the document]
Comment (chenrui333) [reply Formula/v/vercel-cli.rb:38]: it does not break anything, it is just the fsevents.node is only for x86 (and it is not universal binary that we can simply use `deuniversalize_machos`)

**Comment by chenrui333** [comment]
rebased after #151438

**Review by p-linnane — APPROVED** [review]

**Pull request merged**